### DISCUSSION TITLE
What is this?

### DISCUSSION THREAD

**Discussion by NuSynth** [kammt/MemeAssembly#66] [Q&A]
I know it's an assembly language. But I'm not understanding how to use it, and what it's limitations are compared to other assembly languages. Would I be able to write a DOS program with it?

**Comment by kammt** [comment]
> But I'm not understanding how to use it

You can find a documentation of all commands and some info about the general usage at https://kammt.github.io/MemeAssembly/#/ (though that documentation does need some work to be fully beginner-friendly)

> and what it's limitations are compared to other Assembly languages

This assembly language does have support for most of the basic instructions, but I'd consider it to be more *minimalistic*. What it can be used for however is
a) writing small functions that can be called from e.g. C (some example for that would be Stalinsort or rot13, which you can find in the examples-directory) and
b) writing basic programs (e.g. is_nice)

TL;DR: It's a joke programming language which can actually be used in a lot of instances (but not DOS programs)
